frontend/src/entities/dataset/create/dataset-create.tsx

```diff
@@ -127,17 +130,21 @@ export function DatasetCreate () {
             options.push({ label: initCap(DatasetType.PROJECT), value: DatasetType.PROJECT });
         }
 
-        if (groups && groups.length > 0) {
-            const groupLabels: { label: string; value: string }[] = [];
-            groups.map((group, index) => {
-                groupLabels.push({ label: group.name, value: `group${index}`});
-            });
-            options.push({ label: 'Groups', options: groupLabels});
+        if (groups) {
+            options.push({ label: initCap(DatasetType.GROUP), value: DatasetType.GROUP });
         }
 
         return options;
     }
 
+    function generateGroupOptions () {
+        const groupOptions: { label: string; value: string }[] = [];
```

Comment:
Can we just use `SelectProps.Options[]` instead?

Reply:
Seems crazy enough to work